module/zfs/dmu.c

```diff
@@ -738,7 +738,12 @@ dmu_prefetch(objset_t *os, uint64_t object, int64_t level, uint64_t offset,
 	}
 
 	if (nblks != 0) {
+		uint64_t maxnblks;
+		uint64_t maxblkid = dn->dn_maxblkid;
 		blkid = dbuf_whichblock(dn, level, offset);
+		maxnblks = maxblkid + 1 - blkid;
```

Comment:
It looks like the above 3 new lines could be simplified to.

```suggestion
                uint64_t maxnblks = dn->dn_maxblkid + 1 - blkid;
```

Reply:
Yeah